### PR TITLE
Fixed Icons, license, and .Net versions supported

### PR DESCRIPTION
**Pull request by darcythomas**

### CODE REVIEW

File: Mindscape.Raygun4Net.AspNetCore/Mindscape.Raygun4Net.AspNetCore.csproj [hunk 1]
```diff
@@ -9,25 +9,36 @@
...
+    <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="2.2.0" />
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="1.1.2" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
-    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="5.0.0" />
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.1'">
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Extensions" Version="2.2.0" />
+    <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="1.1.2" />
+    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.2.0" />
```
Comment (QuantumNightmare) [inline Mindscape.Raygun4Net.AspNetCore/Mindscape.Raygun4Net.AspNetCore.csproj:37]: Just wondering why this dependency was added

**Review by QuantumNightmare — APPROVED** [review]
Looks good, just added one question for my understanding. Also, we didn't want to use a colored icon? (A colored icon appears on the NuGet site, but this monochromatic one appears in Visual Studio. Can barely see it when using the VS dark theme).

File: Mindscape.Raygun4Net.AspNetCore/Mindscape.Raygun4Net.AspNetCore.csproj (hunk 1 above)
Comment (darcythomas) [reply Mindscape.Raygun4Net.AspNetCore/Mindscape.Raygun4Net.AspNetCore.csproj:37]: I guess I mis-read this. https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0#when-to-target-net50-or-net60-vs-netstandard.

Re reading I guess targeting 2.0 and .net5-6 would be best

**Comment by darcythomas** [comment]
> Looks good, just added one question for my understanding. Also, we didn't want to use a colored icon? (A colored icon appears on the NuGet site, but this monochromatic one appears in Visual Studio. Can barely see it when using the VS dark theme).


I just took the icon which was in the iconlink, downloaded and added that.

File: Mindscape.Raygun4Net.AspNetCore/Mindscape.Raygun4Net.AspNetCore.csproj (hunk 1 above)
Comment (darcythomas) [reply Mindscape.Raygun4Net.AspNetCore/Mindscape.Raygun4Net.AspNetCore.csproj:37]: Hmmm. But this seems to say https://docs.microsoft.com/en-us/dotnet/standard/net-standard?tabs=net-standard-2-0#net-standard-not-deprecated both would be the way to go

Comment (QuantumNightmare) [reply Mindscape.Raygun4Net.AspNetCore/Mindscape.Raygun4Net.AspNetCore.csproj:37]: Yeah, should be fine to target 2.1, was just wondering why the Abstractions package was also added. Perhaps the IDE added this for you.

**Comment by QuantumNightmare** [comment]
Not sure where the colored one in the NuGet site is coming from. Just thinking while you're in this space, could optionally confirm with a designer which icon we want to use.

**Comment by darcythomas** [comment]
Updated Icon (correct color) and removed un-need package imports

**Review by QuantumNightmare — APPROVED** [review]
Thanks for the changes :+1:

**Comment by darcythomas** [comment]
Tested against targets:

<html>
<body>


.NET and .NET Core | 1.0, 1.1, 2.0, 2.1, 2.2, 3.0, 3.1, 5.0, 6.0
-- | --
.NET Framework  | 4.6.1 2, 4.6.2, 4.7, 4.7.1, 4.7.2, 4.8


</body>
</html>

**Pull request merged**